_posts/2021-11-09-controlled-space-egress.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+date: November 09, 2021
+title: "New Controlled Space Egress"
+excerpt: The cloud.gov platform is releasing new space types to help customers better control app egress.
+---
+
+To better help our customers control egress from their apps, we are offering additional space types for your organization with new application security group (ASG) rules. These new ASG rules allow space egress to be more locked down to minimize the impacts of possible data exfiltration. Currently, all cloud.gov org spaces allow egress from an app to the open internet, our brokered services ([AWS RDS](https://cloud.gov/docs/services/relational-database/), [AWS Elasticache Redis](https://cloud.gov/docs/services/aws-elasticache/), [AWS Elasticsearch](https://cloud.gov/docs/services/aws-elasticsearch/)), and [internal routes](https://docs.cloudfoundry.org/devguide/deploy-apps/routes-domains.html#internal-routes) to other apps within your organization. Our additional space ASG offerings will now include `public-egress`, `restricted-egress`, and `closed-egress`. These ASG rules will apply to all apps running within a certain space type and will allow you to group apps based on functionality.
+
+Here are the following rules associated to the different space ASG types:
+
+- ### `public-egress`
+  - Requests being executed from within the app can successfully be sent to the open internet, our brokered services, and other internal routes you have created in your organization.
+  - Your app can make requests to third party APIs
+  - Your app can connect to our data services
+
+- ### `restricted-egress`
+  - Requests being executed from within the app can only successfully be sent to our brokered services or other internal routes you have created in your organization.
```

Comment:
should we call out here that S3 is excluded?

Reply:
I realized you spelled out which services you're talking about above, so I think we're good